scripts/ctool

```diff
@@ -27,7 +27,7 @@ Usage: ${0##*/} mode [options]
   subcommands:
    run-container                [--destroy] [--name=] source [command]
    wait-for-boot [--container=]
-   execute       [--container=] [--dir=] [--user=] command
+   exec|execute  [--container=] [--dir=] [--user=] [command]
```

Comment:
Don't [ ] mean optional?  I figured --container and command would be required.

Reply:
--container is not required:

    $ ctool execute --dir=/var ls
    backups  crash	local  log   metrics  run   spool
    cache	 lib	lock   mail  opt      snap  tmp

    $ ctool execute --user=root ls /root | sed 's/^/OH /; s/$/!/'
    OH snap!